WordPress/src/main/java/org/wordpress/android/datasets/ReaderBlogTableWrapper.kt

```diff
@@ -11,4 +11,18 @@ class ReaderBlogTableWrapper
     fun isNotificationsEnabled(blogId: Long): Boolean = ReaderBlogTable.isNotificationsEnabled(blogId)
     fun setNotificationsEnabledByBlogId(blogId: Long, isEnabled: Boolean) =
             ReaderBlogTable.setNotificationsEnabledByBlogId(blogId, isEnabled)
+    fun getReaderBlog(blogId: Long, feedId: Long): ReaderBlog? {
+        return if (feedId != 0L) {
+            ReaderBlogTable.getFeedInfo(feedId)
+        } else {
+            ReaderBlogTable.getBlogInfo(blogId)
+        }
+    }
+    fun isSiteFollowed(blogId: Long, feedId: Long): Boolean {
+        return if (feedId != 0L) {
+            ReaderBlogTable.isFollowedFeed(feedId)
+        } else {
+            ReaderBlogTable.isFollowedBlog(blogId)
+        }
+    }
```

Comment:
This `if else` statement is duplicated now, I wonder if we could handle this in the DB level by using an SQL `SELECT * FROM tbl_blog_info WHERE blog_id=? AND feed_id=?` expression. The pair of `blog_id` and `feed_id` should be unique, right?
Any thoughts on this?